src/components/accordion/accordion.ts

```diff
@@ -111,7 +111,13 @@ export class Accordion extends LitElement {
     return html`
       <div class="kd-accordion">
         <div class="toggle-container">
-          <a href="javascript:void(0)" @click="${this._toggleExpandAll}">
+          <a
+            href="javascript:void(0)"
+            @click="${this._toggleExpandAll}"
+            aria-label=${this._allOpenState

```

Comment:
This aria-label also seems redundant because there is visible text.

Reply:
This is for screen readers. Robert put an issue for this

Reply:
[Screen Recording 2024-08-09 at 6.24.24 PM.mov](https://dev.azure.com/Kyndryl/f7f7ab25-06ec-43dc-902d-dee2e157cebd/_apis/wit/attachments/10557fe4-4edb-4f43-88cd-de824655e5a5?fileName=Screen%20Recording%202024-08-09%20at%206.24.24%E2%80%AFPM.mov&download=true)

Reply:
But why won't they read it? The text is present and visible.